pkg/repositories/repository_get_all_suite_test.go

```diff
@@ -16,7 +16,7 @@ type RepositoryGetAllTestSuite struct {
 	testFunc testGetAllFunc
 
 	expectedScanCalls    int
-	expectedScannedProps []interface{}
+	expectedScannedProps [][]interface{}
```

Comment:
Enhanced this test suite (and others) to allow registering the props passed to successive calls to `scan`.